### PR TITLE
chore: changes for Open Pet Food Facts to the nutrient list

### PR DESCRIPTION
**Pull request by teolemon**
changes to Food.pm for Open Pet Food Facts to the nutrient list

### CODE REVIEW

File: lib/ProductOpener/Food.pm [hunk 1]
```diff
@@ -3843,7 +3843,181 @@ sub mmoll_to_unit {
...
+	de => "Zellulose",
+	el => "Κυτταρίνη",
+	en => "Cellulose",
+	eo => "Celulozo",
+	es => "Celulosa",
+	et => "Tselluloos",
+	eu => "Zelulosa",
+	fa => "سلولز",
+	fi => "Selluloosa",
+	fr => "Cellulose", "Cellulose brute",
```
Comment (stephanegigandet) [inline lib/ProductOpener/Food.pm:3869]:
```suggestion
	fr => "Cellulose",
	fr_synonyms => ["Cellulose brute"],
```

File: lib/ProductOpener/Food.pm [hunk 2]
```diff
@@ -3843,7 +3843,181 @@ sub mmoll_to_unit {
...
+	ay => "Qhilla",
+	bat_smg => "Pelenā",
+	bg => "Пепел",
+	br => "Ludu",
+	ca => "Cendra",
+	cs => "Popel",
+	da => "Aske",
+	de => "Asche",
+	diq => "Wele",
+	en => "Ash", "Crude ash",
```
Comment (stephanegigandet) [inline lib/ProductOpener/Food.pm:3938]: there can only be one hash value, use en_synonyms => [ "syn 1", "syn 2"],

**Comment by stephanegigandet** [comment]
Side effect: those new entries will be visible on OFF, e.g. in the suggest as you type when adding new nutrients.

File: lib/ProductOpener/Food.pm [hunk 3]
```diff
@@ -3843,7 +3843,182 @@ sub mmoll_to_unit {
...
```
Comment (teolemon) [inline lib/ProductOpener/Food.pm:3939]:
```suggestion
	en => "Ash",
	en_synonyms => ["Crude ash"],
```

**Comment by teolemon** [comment]
I can leave them commented out if you wish.
Having them as suggestions is not much a risk though.

File: lib/ProductOpener/Food.pm [hunk 4]
```diff
@@ -3843,7 +3843,183 @@ sub mmoll_to_unit {
...
+	fi => "Selluloosa",
+	fr => "Cellulose",
+	fr_synonyms => ["Cellulose brute"],
+	ga => "Ceallalós",
+	gl => "Celulosa",
+	he => "תאית",
+	hi => "सेलुलोस",
+	hr => "Celuloza",
+	ht => "Seliloz",
+	hu => "Cellulóz,
```
Comment (VaiTon) [inline lib/ProductOpener/Food.pm:3877]:
```suggestion
	hu => "Cellulóz",
```

File: lib/ProductOpener/Food.pm [hunk 5]
```diff
@@ -3843,7 +3843,183 @@ sub mmoll_to_unit {
 	carnitine => {
 		en => "Carnitine",
 		fi => "Karnitiini"
-	}
+	},
+"cellulose" => {
```
Comment (VaiTon) [inline lib/ProductOpener/Food.pm:3847]:
```suggestion
cellulose => {
```

**Comment by teolemon** [comment]
189 distributions installed
syntax error at lib/ProductOpener/Food.pm line 3953, near "nb"
Compilation failed in require at lib/ProductOpener/Tags.pm line 154.
BEGIN failed--compilation aborted at lib/ProductOpener/Tags.pm line 154.
Compilation failed in require at lib/ProductOpener/Display.pm line 127.
BEGIN failed--compilation aborted at lib/ProductOpener/Display.pm line 127.
Compilation failed in require at lib/startup_apache2.pl line 67.
BEGIN failed--compilation aborted at lib/startup_apache2.pl line 67.

File: lib/ProductOpener/Food.pm [hunk 6]
```diff
@@ -3843,7 +3843,183 @@ sub mmoll_to_unit {
...
+	ku => "Seluloz",
+	lt => "Celiuliozė",
+	lv => "Celuloze",
+	mk => "Целулоза",
+	ml => "സെല്ലുലോസ്",
+	mr => "सेल्युलोज",
+	ms => "Selulosa",
+	my => "ဆဲလျူလို့",
+	nl => "Cellulose",
+	nn => "Cellulose"
```
Comment (teolemon) [inline lib/ProductOpener/Food.pm:3897]:
```suggestion
	nn => "Cellulose",
```

**Pull request closed without merging**